surfer/viz.py

```diff
@@ -1115,27 +1140,30 @@ def time_label(x):
             data["time"] = time
             data["time_idx"] = 0
             y_txt = 0.05 + 0.05 * bool(colorbar)
-        else:
-            initial_time_index = None
 
         surfs = []
         bars = []
+        glyphs = []
         views = self._toggle_render(False)
+        vector_alpha = alpha if vector_alpha is None else vector_alpha
         for brain in self._brain_list:
             if brain['hemi'] == hemi:
-                s, ct, bar = brain['brain'].add_data(
-                    array, mlab_plot, min, max, thresh, lut, colormap, alpha,
-                    colorbar, layer_id)
+                s, ct, bar, gl = brain['brain'].add_data(
+                    array, min, mid, max, thresh, lut, colormap, alpha,
+                    colorbar, layer_id, smooth_mat, magnitude, magnitude_max,
+                    scale_factor, vertices, vector_alpha)
                 surfs.append(s)
                 bars.append(bar)
-                if array.ndim == 2 and time_label is not None:
+                glyphs.append(gl)
```

Comment:
Wouldn't it be simpler to just set `glyphs` to `None` or `()` instead of creating list of `None` for non-vector data?